src/addons/mod/scorm/classes/data-model-12.ts

```diff
@@ -255,6 +255,11 @@ export class AddonModScormDataModel12 {
      * @param userData The user default data.
      */
     protected init(userData: AddonModScormUserDataMap): void {
+        if (this.scorm.scormStandard !== 0) {
```

Comment:
```suggestion
        if (this.scorm.scormStandard === 0 || this.scorm.scormStandard === undefined) {
```
The value of the variables should be changed if the setting is disabled (=== 0). Also, if we don't have the setting because the Moodle site doesn't support returning it I think we decided to always apply the 64000 value so affected SCORMs can work in the app when using old sites.

Reply:
What do you think about use `!this.scorm.scormStandard` directly?
Maybe this is cleaner and cover nullish too.

Reply:
Sure, sounds good to me.